source/manual/business-readiness-finder.html.md

```diff
@@ -0,0 +1,71 @@
+
+We added the [ability to 'pin' content items](https://github.com/alphagov/content-tagger/pull/884) (via the content-tagger) to the top of finder results. This feature is not currently in use and could be removed. The corresponding [pinning code in the support app](https://github.com/alphagov/support/pull/574) should also be removed.
+
+We also added the ability to [sort by topic](https://github.com/alphagov/finder-frontend/pull/856). This is only applicable to the business readiness finder, but is not a candidate for deletion, as it's been proven to be a useful feature.
+
+The finder uses the search-api to find and filter results. Whilst standard finders get more granular as you select values in different facets (the number of results goes down), the business finder gets more _broad_ (number of results goes up).
+
+Standard finders have facets under `details.facets` in their [JSON structure](https://www.gov.uk/api/content/cma-cases). This finder instead has [facets under `links.facet_group`](https://www.gov.uk/api/content/find-eu-exit-guidance-business). The default way of filtering by links is to use 'OR logic', as opposed to the 'AND logic' exhibited in other finders.
+
+We did have [code to force the links into an 'AND logic' behaviour](https://github.com/alphagov/search-api/pull/1501), but this is no longer being used and should be removed.
+
+### Email signup page
+
+We added two features to the email signup page:
+
+1. Users can [sign up to multiple `facet_values`](https://www.gov.uk/find-eu-exit-guidance-business/email-signup).
+2. Email subscription frequency has been [changed to "Daily" by default](https://github.com/alphagov/finder-frontend/pull/1104), in response to user research.
+
+Currently only the business finder is making use of these features, but we believe other finders could use these in the future. This is documented further in the [finder-frontend repository](https://github.com/alphagov/finder-frontend/blob/master/docs/email-signup.md).
+
+### Support form
+
+We've got a support form in the support app, which allows editors to request changes to the business finder. It currently [displays options for pinning code](https://github.com/alphagov/support/pull/574), which should be removed when the corresponding behaviour is removed from content-tagger.
+
+It's also worth noting that the question answers are driven by the facet values in most cases, but [hard-coded to 'yes'/'no' values in others](https://github.com/alphagov/support/blob/master/app/views/eu_exit_business_readiness_requests/_request_details.html.erb#L87).
```

Comment:
I'm not sure if this section is needed. The details here are already in the PR description and feel a bit too granular to be included here.